OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -121,6 +155,13 @@ open class OptimoveConfigBuilder: NSObject {
 
         return self
     }
+    
+    @objc @discardableResult public func enableDeepLinking(cname: String? = nil, _ objcHandler: @escaping (NSDictionary) -> Void) -> OptimoveConfigBuilder {
+        _objCdeepLinkHandler = objcHandler
```

Comment:
Is it necessary to store the objc handler as a field, can we just capture it in the closure scope and save adding a field for it?

Reply:
i think it is better for memory handling purposes

Reply:
Oh cool, could you explain why / share a reference so I can learn more about this?

Reply:
because swift arc (automatic count reference) will keep hold of the class (it will be strong referenced) and it could lead to retain cycle